program/src/error.rs

```diff
@@ -146,6 +146,10 @@ pub enum LidoError {
     /// There is a validator that has more stake than the selected one.
     #[error("ValidatorWithMoreStakeExists")]
     ValidatorWithMoreStakeExists = 37,
+
+    /// The instance has no validators.
+    #[error("EmptySetOfValidators")]
+    EmptySetOfValidators = 38,
```

Comment:
We already have `NoActiveValidators = 27`. That name is outdated, we don’t need an “active” stake account for the validator, `EmptySetOfValidators` is a better name. But we should merge the two.

Reply:
Great, changed it to `EmptySetOfValidators`